core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -74,14 +74,6 @@ protected[controller] sealed class WebApiDirectives private (prefix: String) {
 protected[controller] object WebApiDirectives {
```

Comment:
i suggest you remove this entire class - and change the constructor for WebApiDirectives:

```
protected[controller] sealed class WebApiDirectives private (prefix: String = "__ow_") {
```